lock/src/main/java/com/auth0/android/lock/PasswordlessLockActivity.java

```diff
@@ -0,0 +1,236 @@
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package com.auth0.android.lock;
+
+
+import android.content.Intent;
+import android.os.Build;
+import android.os.Bundle;
+import android.os.Handler;
+import android.support.annotation.Nullable;
+import android.support.v4.content.LocalBroadcastManager;
+import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.LinearLayout;
+
+import com.auth0.Auth0Exception;
+import com.auth0.android.lock.enums.PasswordlessMode;
+import com.auth0.android.lock.events.PasswordlessLoginEvent;
+import com.auth0.android.lock.views.LockProgress;
+import com.auth0.android.lock.views.PasswordlessFormView;
+import com.auth0.authentication.AuthenticationAPIClient;
+import com.auth0.authentication.AuthenticationRequest;
+import com.auth0.authentication.PasswordlessType;
+import com.auth0.authentication.result.Authentication;
+import com.auth0.callback.BaseCallback;
+import com.auth0.request.ParameterizableRequest;
+import com.squareup.otto.Bus;
+import com.squareup.otto.Subscribe;
+
+import java.util.HashMap;
+
+public class PasswordlessLockActivity extends AppCompatActivity {
+
+    private static final String TAG = PasswordlessLockActivity.class.getSimpleName();
+
+    private Options options;
+    private Handler handler;
+    private Bus lockBus;
+    private LinearLayout rootView;
+    private LockProgress progress;
+    private PasswordlessFormView passwordlessForm;
+    private String lastPasswordlessEmailOrNumber;
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        if (!isLaunchConfigValid()) {
+            finish();
+            return;
+        }
+
+        lockBus = new Bus();
+        lockBus.register(this);
+        handler = new Handler(getMainLooper());
+
+        setContentView(R.layout.com_auth0_lock_activity_lock);
+        progress = (LockProgress) findViewById(R.id.com_auth0_lock_progress);
+        progress.showResult("");
+        rootView = (LinearLayout) findViewById(R.id.com_auth0_lock_content);
+        initLockUI();
+    }
+
+    private boolean isLaunchConfigValid() {
+        options = getIntent().getParcelableExtra(Lock.OPTIONS_EXTRA);
+        if (options == null) {
+            Log.e(TAG, "You need to specify the com.auth0.android.lock.Options in the Lock.OPTIONS_EXTRA of the Intent for PasswordlessLockActivity to launch. " +
+                    "Use com.auth0.android.lock.PasswordlessLock.Builder to generate one.");
```

Comment:
No need for full package name, just `PasswordlessLock.Builder` will do. Also let's clarify that they need to create the intent using the builder